schema-yield.graphql

```diff
@@ -172,8 +175,8 @@ type FinancialsDailySnapshot @entity {
   " Gross revenue for the protocol (revenue claimed by protocol). Examples: AMM protocol fee (Sushi’s 0.05%). OpenSea 10% sell fee. "
   protocolSideRevenueUSD: BigDecimal!
 
-  " Fees paid by the users. e.g. 0.30% of swap fee in Sushiswap "
-  feesUSD: BigDecimal!
+  " Total revenue accrued by the protocol. e.g. 0.30% of swap fee in Sushiswap, all yield generated by Yearn. "
+  totalRevenueUSD: BigDecimal!
```

Comment:
How is `totalRevenueUSD` different from `protocolSideRevenueUSD` ??

Reply:
`totalRevenueUSD` is literally all money coming in. In the case of yield aggregators, revenue are split between:
- Supply side (liquidity providers who deposit)
- Strategists
- Partners (in the case of Yearn's partnership program)
- Protocol themselves (all fees collected by the protocol after subtracting the strategist portion and the partner portion)

`protocolSideRevenueUSD` would correspond to the last item above